### PR TITLE
feat: integrate AntV G6 for graph visualization

### PR DESCRIPTION
**Pull request by yvonneyx**
It seems that your project involves some works of graph visualization. We would like to suggest considering the use of AntV G6, as it might enhance the overall user experience and visual quality.

Google Chrome 2024-08-13 17 10 11
Google Chrome 2024-08-13 17 10 21

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,1199 +1,2062 @@
...
+          //       res += `<div>Website: <a href=${_website} target="_blank">${_website}</a></div>`;
+          //     }
+          //     return res;
+          //   },
+          //   enable: (e) => ["node", "canvas"].includes(e.targetType),
+          //   enterable: true,
+          //   position: "right",
+          // },
+        ],
+      });
```
Comment (hustcc) [inline index.html:2005]: 删除。

File: index.html [hunk 2]
```diff
@@ -1,1199 +1,2062 @@
...
-            {id: 'Zorro', _paper: 'Zorro: Zero-Cost Reactive Failure Recovery in Distributed Graph Processing'},
-            {id: 'faimGraph', _paper: 'faimGraph: High Performance Management of Fully-Dynamic Graphs under tight Memory Constraints on the GPU'},
-            {id: 'iGraph', _paper: 'iGraph: an incremental data processing system for dynamic graph'},
-            {id: 'iMapReduce', _paper: 'iMapReduce: A Distributed Computing Framework for Iterative Computation'},
-            {id: 'iPregel', _paper: 'iPregel: A Combiner-Based In-Memory Shared Memory Vertex-Centric Framework'},
-            {id: 'iTurboGraph', _paper: 'iTurboGraph: Scaling and Automating Incremental Graph Analytics'},
-            {id: 'xDGP', _paper: 'xDGP: A Dynamic Graph Processing System with Adaptive Partitioning'},
-        ]);
+      // create nodes
+      const nodes = [
```
Comment (hustcc) [inline index.html:133]: 数据可以不格式化，减少行数。

**Comment by hustcc** [comment]
1. readme 中的截图需要换一下
2. PR 描述中关联一下 Issue

**Review by fanzhidongyzby — APPROVED** [review]
LGTM

**This is an awesome feature !**

**Pull request merged**